test/wallaby/browser/dialog_test.exs

```diff
@@ -0,0 +1,208 @@
+defmodule Wallaby.Browser.DialogTest do
+  use Wallaby.SessionCase, async: true
+
+  setup %{session: session} do
+    page = visit(session, "dialogs.html")
+    {:ok, %{page: page}}
+  end
+
+  describe "accept_dialogs/1" do
+    test "accept window.alert", %{page: page} do
+      result = page
+      |> accept_dialogs()
+      |> click(Query.link("Alert"))
+      |> find(Query.css("#result"))
+      |> Element.text()
```

Comment:
I actually prefer the style in the second example, but tried to stick to the way existing tests were written, because those that I had looked at were like this. As I read some more code and saw different styles being mixed, I'm not so sure which is the preferred style. Let @keathley decide 😉